src/public/resolvers/index.ts

```diff
@@ -14,6 +14,10 @@ export const resolvers = {
   CorpusItem: {
     __resolveReference: getCorpusItem,
   },
+  // Allow the `SavedItem` to resolve the corpus item
+  SavedItem: {
+    corpusItem: getSavedCorpusItem,
```

Comment:
@collectedmind needed to use a separate resolver, because I forgot the reference resolvers (the one above this line) pass in different parameters, so we needed to create a slightly modified one for regular fields.